src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -208,7 +209,7 @@ bool emberAfOperationalCredentialsClusterRemoveFabricCallback(chip::app::Command
     EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
     AdminPairingInfo * admin;
     AdminId adminId;
-    CHIP_ERROR err = CHIP_NO_ERROR;
+    CHIP_ERROR err;
```

Comment:
nit: if you are changing this line, you can update line 220 instead and avoid c-style delcataion of everything at top. Something like:

```cpp
VerifyOrExit(
    GetGlobalAdminPairingTable().Delete(adminId) == CHIP_NO_ERROR,
    status = EMBER_ZCL_STATUS_FAILURE);
```

Reply:
That change was unintentional. I have reverted it to the old code.